test/unittests/commands/xmlupload/test_read_validate_xml_file.py

```diff
@@ -5,6 +5,8 @@
     _check_if_salsah_targets_exist,
 )
 
+# pylint: disable=missing-function-docstring
```

Comment:
All functions in this module have docstrings. And pylint passes on main branch, so this change should be reverted